docs/en/ingest-management/elastic-agent/configuration/providers/kubernetes-provider.asciidoc

```diff
@@ -0,0 +1,429 @@
+[[kubernetes-provider]]
+= Kubernetes Provider
+
+Provides inventory information from Kubernetes.
+
+
+[discrete]
+== Provider configuration
+
+[source,yaml]
+----
+providers.kubernetes:
+  node: ${NODE_NAME}
+  scope: node
+  #kube_config: /Users/elastic-agent/.kube/config
+  #sync_period: 600s
+  #cleanup_timeout: 60s
+  resources:
+    pod:
+      enabled: true
+----
+
+`node`:: (Optional) Specify the node to scope {agent} to in case it
+cannot be accurately detected when running {agent} in host network
+mode.
```

Comment:
We could state the default value here and to the rest of the settings we haven't already done so